tests/unit/sys/test_options.cxx

```diff
@@ -32,6 +32,20 @@ TEST_F(OptionsTest, IsSetDefault) {
   ASSERT_FALSE(options.isSet("default_value"));
 }
 
+TEST_F(OptionsTest, IsSection) {
+  Options options;
+
+  // make sure options is initialized as a section
+  options["testkey"] = 1.;
+
+  ASSERT_TRUE(options.isSection(""));
```

Comment:
Probably should also check that calling it without arguments has the same effect.